include/boost/phoenix/core/actor.hpp

```diff
@@ -143,6 +143,9 @@ namespace boost { namespace phoenix
             >::type
             expr_type;
 
+        BOOST_DEFAULTED_FUNCTION(actor() BOOST_NOEXCEPT, {});
```

Comment:
This breaks placeholders podness on C++03.

Reply:
> This breaks placeholders podness on C++03.

Do you prefer I use some `#ifndef BOOST_NO_CXX11_DEFAULTED_FUNCTIONS` instead ?

Reply:
I do not know. I am not a maintainer of the library. I just pointed that the change brings more trouble than it solves.

Reply:
I think those should be trivial as much as possible.

Reply:
I have updated the code, it now uses `#ifndef BOOST_NO_CXX11_DEFAULTED_FUNCTIONS`